models/ROMS_rutgers/model_mod.f90

```diff
@@ -472,6 +476,8 @@ subroutine model_interpolate(state_handle, ens_size, location, qty, expected_obs
   expected_obs = sensible_temp(expected_T, expected_S, pdbar) 
 endif
 
+if (convert_salinity_to_kgkg .and. qty == QTY_SALINITY) expected_obs = expected_obs / CONCENTRATION_TO_PPT
```

Comment:
good. This is the right location. We don't want to do that prior to the sensible temperature computation (we need salinity to be in PSU there).